ui/src/main/webapp/package.json

```diff
@@ -60,23 +68,13 @@
     "showdown": "^1.5.1",
     "text-loader": "^0.0.1",
     "webpack-merge": "^0.15.0",
-    "zone.js": "^0.7.6",
-    "d3": "4.4.0",
-    "d3-transition": "1.0.3",
-    "d3-selection-multi": "1.0.0",
-    "ngx-charts": "^3.0.2",
-    "angular2-moment": "^1.1.0"
+    "zone.js": "^0.7.6"
```

Comment:
+1 - I actually reordered them accidentally by running yard add. It'd be nice to go ahead and get this done in a separate PR.